Sources/Kitura/staticFileServer/StaticFileServer.swift

```diff
@@ -77,13 +78,24 @@ open class StaticFileServer: RouterMiddleware {
         /// - Parameter redirect: an indication whether to redirect to trailing
         /// "/" when the requested path is a directory.
         /// - Parameter cacheOptions: cache options for StaticFileServer.
+        /// - Parameter defaultIndex: A default index, like "/index.html", to be served if the
+        /// requested path is not found. This is intended to be used by single page applications
+        /// that wish to fallback to a default index when a requested path is not found.
```

Comment:
Given the addition of the check for a file request, is it worth adding something like this?
>when a requested path is not found, and where that path is not a file request.